src/DataDog.Tracing/DataDog.Tracing.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard1.4</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Is this upgrade actually necessary? It's usually better to target as low a netstandard version as you can so that more people can use it.

Reply:
Yes, DbCommand and DbConnection classes are not implemented in 1.4.

Reply:
ok